server/services/personService.ts

```diff
@@ -31,4 +32,17 @@ export default class PersonService {
     const rosh = await personClient.oasysRosh(crn)
     return rosh
   }
+
+  async getRoshRisks(token: string, crn: string): Promise<RoshRisks> {
+    const personClient = this.personClientFactory(token)
+
+    const risks = await personClient.risks(crn)
+
+    return {
+      ...risks.roshRisks?.value,
+      lastUpdated: risks.roshRisks?.value?.lastUpdated
```

Comment:
should the service be responsible for transforming the date here, or do we think better for it to just return the data we get back from the client? I'm wondering if makes more sense for the levels closer to the UI to decide how a date is rendered. Also feel a bit uncomfortable about setting it to an empty string when there are no `roshRisks`...

Reply:
thanks - I agree this needed changing and have done so as discussed :)